.github/scale-config.yml

```diff
@@ -18,6 +18,12 @@
 #     is_ephemeral: true
 
 runner_types:
+  # mainly used for ciflow-should-run, not made to run any serious tests
+  linux.large:
+    instance_type: c5.large
+    os: linux
+    disk_size: 10
+    is_ephemeral: false
```

Comment:
Should we still add a max_available even tho in theory it could be a really large number?

Reply:
`max_available` doesn't actually do anything right now so we can leave it out